File: 2st/Riyeon/배열 비교하기.java
class Solution {
    public int solution(int[] arr1, int[] arr2) {
        int answer = 0;
        
        if(arr1.length == arr2.length){
            int a1=0, a2=0;
            for(int i=0; i<arr1.length; i++){
                a1 += arr1[i];
                a2 += arr2[i];
            }
            if(a1 == a2)
                return 0;
            else
                return a1>a2? 1 : -1;
        }else
            return arr1.length > arr2.length? 1 : -1;
    }
}

File: 2st/Riyeon/배열 비교하기.py
def solution(arr1, arr2):
    answer = 0
    if len(arr1) == len(arr2):
        if sum(arr1) != sum(arr2):
            answer = 1 if sum(arr1) > sum(arr2) else -1
    else:
        answer = 1 if len(arr1) > len(arr2) else -1
    return answer